spec/mail/thread.mdown

```diff
@@ -14,6 +14,7 @@ A **Thread** object has the following properties:
   The ids of the messages in the thread, sorted such that:
 
     - Any message with `isDraft == true` and an *inReplyToMessageId* property that corresponds to another message in the thread comes immediately after that message in the sort order.
+    - Directly after, any message with `isDraft == true` and an *fowardingMessageId* property that corresponds to another message in the thread comes immediately after that message in the sort order.
```

Comment:
On GMail for example forwarded messages are mixed with replied messages, all being sort by date.
Seems better, no?